### ISSUE TITLE
WCAG 2.1: Target app highlighter is off-screen on 3 sides at 200% zoom

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#767]
**Describe the bug**
Target app highlighter is not visible on 3 sides at 200% zoom, when ```What to select``` is set to ```Entire app```

**To Reproduce**
Adjust the following display settings in Windows :
- Display resolution to ```1280 * 1024```
- Scaling size to ```200%```

1.  Open AI-Windows
2. Set ```What to select``` to ```Entire app```
3. Hover on any windows app opened in maximized mode
4. Observe the highlighter is off-screen on 3 sides and the beaker is not completely visible

**Expected behavior**
When the app content has re-flowed, there should not loss of information or functionality

**Actual behavior**
The highlighter is not visible on all sides

**Screenshots or .GIF**
![3](https://user-images.githubusercontent.com/22068434/79996799-b4551d00-846d-11ea-93ca-03fc9468888e.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, Version1903
 - Accessibility Insights for Windows Version: Production Release: v1.1.1198.01

**Comment by adarshrema** [comment]
From internal testing for accessibility. To be combined with results from planned HC testing.
